.github/workflows/test-suite.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           tools: composer:v2
           php-version: "${{ matrix.php-version }}"
           ini-values: memory_limit=-1
+          extensions: :psr
```

Comment:
I'm honestly not sure why suddently psr extension become installed but it was breaking the testsuite, disabled according to [docs](https://github.com/marketplace/actions/setup-php-action#heavy_plus_sign-php-extension-support)